code/modules/food_and_drinks/food/foods/junkfood.dm

```diff
@@ -121,6 +121,16 @@
 	user.put_in_hands(L)
 	qdel(src)
 
+/obj/item/food/chocolate_bar
```

Comment:
I recommend changing the path name since we currently have "/obj/item/food/chocolatebar", and "/datum/chemical_reaction/chocolate_bar".
Could be /delux_chocolate_bar or something like this.

Reply:
Change made!